### ISSUE TITLE
scikit-learn version clarification

### ISSUE THREAD

**Issue by andrisecker** [pafoster/pyitlib#17]
Hi @pafoster, I was wondering about the reasoning behind limiting `scikit-learn` version to `<=0.24`. It's causing issues to use this package together with other ones that rely on the latest version of Python an `scikit-learn` versions above `1.`

**Comment by ethanpk** [comment]
Yes, the package runs with any version of Scikit-learn. Can you update the source code, building wheel for scikit-learn is creating issues

**Comment by Jasperhino** [comment]
+1 on this

**Comment by ggmirandac** [comment]
+2 on this :(

**Comment by pafoster** [comment]
I've removed the limitation.
